src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/PostTaskInitiateByIdControllerTest.java

```diff
@@ -220,11 +310,47 @@ public void should_return_a_503_if_task_already_initiated() {
             .body("status", equalTo(503))
             .body("detail", equalTo(
                 "Database Conflict Error: The action could not be completed because "
-                    + "there was a conflict in the database."));
+                + "there was a conflict in the database."));
 
         common.cleanUpTask(taskId);
     }
 
+    @Test
+    public void should_return_a_400_if_no_due_date() {
+        TestVariables taskVariables = common.setupTaskAndRetrieveIds();
+        String taskId = taskVariables.getTaskId();
+        common.setupOrganisationalRoleAssignment(authenticationHeaders);
+
+        ZonedDateTime createdDate = ZonedDateTime.now();
+        String formattedCreatedDate = CAMUNDA_DATA_TIME_FORMATTER.format(createdDate);
+
+        InitiateTaskRequest req = new InitiateTaskRequest(INITIATION, asList(
+            new TaskAttribute(TASK_TYPE, "followUpOverdueReasonsForAppeal"),
+            new TaskAttribute(TASK_NAME, "follow Up Overdue Reasons For Appeal"),
+            new TaskAttribute(TASK_TITLE, "A test task"),
+            new TaskAttribute(TASK_CREATED, formattedCreatedDate),
+            new TaskAttribute(TASK_CASE_ID, taskId)
+        ));
+        //First call
+        Response result = restApiActions.post(
+            ENDPOINT_BEING_TESTED,
+            taskId,
+            req,
+            authenticationHeaders
+        );
+
+        result.then().assertThat()
+            .statusCode(HttpStatus.BAD_REQUEST.value())
+            .contentType(APPLICATION_PROBLEM_JSON_VALUE)
+            .body("type", equalTo(
+                "https://github.com/hmcts/wa-task-management-api/problem/constraint-validation"))
+            .body("title", equalTo("Constraint Violation"))
+            .body("status", equalTo(400))
+            .body("violations[0].field", equalTo("task_due_date"))
```

Comment:
should be .violations[0].field & .violations[0].message as per your slack message :). Need your update